### PR TITLE
LG/DCOS-46836 - DRAFT content for 1.10.10 release notes

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
## Description

https://jira.mesosphere.com/browse/DCOS-46836

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.10/release-notes/1.10.10/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+excerpt:
+---
+
+DC/OS 1.10.10 was released on Jaunary 18, 2019.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.10/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Updated Components in DC/OS 1.10.10
+DC/OS 1.10.9 includes the following:
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Replace 1.10.9 with 1.10.10.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+- DCOS-44238 - You can download the latest version of the `core-cli` for a cluster without specifying a patch version or using automatic installation.
+
+## Docker Integration
+- COPS-4044, DCOS_OSS-4469 - This release changes the logging settings for the `dcos-docker-gc` unit so that any log messages it creates are preserved in the `systemd` journal logging facility on the host system.
+
+## Mesos
+- COPS-4320, DCOS-46814 - After an agent host is rebooted, the forked child process id and `libprocess` process id for executor in the agent's meta directory are obsolete and should not be read. This change to the process identifiers read during agent recovery prevents the container from waiting for a process if those process ids are reused after a reboot. 
+
+  Previously, if you rebooted an agent, the agent would wait for the exit status of its container process id (`pid`) before terminating the executor. If a new process with the same `pid` is spawned after the reboot, the agent recovery might stall waiting for the wrong child process id, blocking the executor termination and updates to its tasks.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Remove the white space in the beginning of this line.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  
+- DCOS-46388 - The master node completes the processing of all authorization results for a `LAUNCH_GROUP` before performing other operations. This change prevents subsequent operations from failing if any authorization request is denied.
+
+- DCOS-46753 - This release improves how failed or discontinued launch operations are handled to ensure container input and output operations are resolved correctly and all file descriptors are closed properly. 
+
+  Previously, if the containerizer launch failed or was discarded after the I/O switchboard server started but before the container process completed execution, the file descriptor used to signal a redirect to the I/O switchboard could fail, preventing the containerizer from completing its clean-up operations. You might see this issue if you have frequent health or readiness checks for containers launching on an agent with heavy processing load.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Remove the white space in the beginning of this line.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+  If rebooting a node does not delete old containers and IP/MAC addresses from `etcd` (which over time can cause pool exhaustion), you should set the `--network_cni_root_dir_persist` agent option in the `config.yaml` file to `true`. Note that changing this flag requires rebooting the agent node or shutting down all container processes running on the node. Because a reboot or shutdown of containers is required, the default value for the `--network_cni_root_dir_persist` agent option is `false`. Before changing this option, you should plan for agent maintenance to minimize any service interruption. If you set this option and reboot a node, you should also unset the `CNI_NETNS` environment variable after rebooting using the CNI plugin `DEL` command so that the plugin cleans up as many resources as possible (for example, by releasing IPAM allocations) and returns a successful response.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Use the format for "Note" as documented in https://docs.google.com/document/d/1wgGiZiZe_8dSOrhmtMuDHmdxAlDXXOjwm4JNAxEUNzw/edit#heading=h.l0rfrv3bmeqq

Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: Senthil: I think the component change log links I have in the 1.10.10 RN are the ones that were in the 1.10.9 RN. We have 1.10.10 and 1.11.9 kind of piling up close together, but can you let me know if the links should change for the 1.10.10 RN?

I'm assuming the download links might not be ready yet ? If they are, I can update that info too ...

File: pages/1.10/release-notes/1.10.10/index.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+- DC/OS 1.10 requires DC/OS CLI 0.5.x.
+- DC/OS CLI 0.5.x adds [multi-cluster support](/1.10/cli/multi-cluster-cli/) with [`dcos cluster`](/1.10/cli/command-reference/dcos-cluster) commands.
+
+
+ Multi-cluster support has a number of consequences:
+
+   - DC/OS CLI 0.4.x and 0.5.x use a different structure for the location of configuration files. DC/OS CLI 0.4.x has a single configuration file, which by default is stored in `~/.dcos/dcos.toml`. DC/OS CLI 0.5.x has a configuration file for each connected cluster, which by default are stored in `~/.dcos/clusters/<cluster_id>/dcos.toml`.
+   - DC/OS CLI 0.5.x introduces the `dcos cluster setup` command to configure a connection to a cluster and log into the cluster.
+   - **Note:**
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Remove the bulleted list for "Note"

File: pages/1.10/release-notes/1.10.10/index.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+### Networking
+- Configurable Spartan upstreams for domains (dnames).
+  You can now configure Spartan to delegate a particular domain (for example, `\*.foo.company.com`) to a particular upstream.
+
+- Increased CNI network support.
+  DC/OS now supports any type of CNI network. [View the documentation](/1.10/networking/virtual-networks/cni-plugins/).
+
+- Edge-LB load balancer. [enterprise type="inline" size="small" /]
+
+  Edge-LB load balances Mesos tasks. Not supported in strict security mode. [View the documentation](/services/edge-lb/0.1/).
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Bulleted list?

File: pages/1.10/release-notes/1.10.10/index.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+## Security
+- DCOS_OSS-4418 - This release includes an upgrade to the Python requests library used in DC/OS to address moderate security vulnerability reports (CVE-2018-18074). The release upgrades the request library from 2.20.1 to 2.21.0.
+
+# About DC/OS 1.10
+
+DC/OS 1.10 includes many new capabilities for operators and expands the collection of Data & Developer Services with a focus on:
+
+- Core DC/OS service continuity - System resilience, IAM scalability & simplified upgrades.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Use **and** instead of &.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+
+  If rebooting a node does not delete old containers and IP/MAC addresses from `etcd` (which over time can cause pool exhaustion), you should set the `--network_cni_root_dir_persist` agent option in the `config.yaml` file to `true`. You should note that changing this flag requires rebooting the agent node or shutting down all container processes running on the node. Because a reboot or shutdown of containers is required, the default value for the `--network_cni_root_dir_persist` agent option is `false`. Before changing this option, you should plan for agent maintenance to minimize any service interruption. If you set this option and reboot a node, you should also unset the `CNI_NETNS` environment variable after rebooting using the CNI plugin `DEL` command so that the plugin cleans up as many resources as possible (for example, by releasing IPAM allocations) and returns a successful response.
+
+## Security
+- DCOS_OSS-4418 - This release includes an upgrade to the Python requests library used in DC/OS to address moderate security vulnerability reports (CVE-2018-18074). The release upgrades the request library from 2.20.1 to 2.21.0.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: This falls under "Notable Changes" category.
`The release upgrades the request library from 2.20.1 to 2.21.0.`

File: pages/1.10/release-notes/1.10.10/index.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+
+- Kubernetes on DC/OS is beta with DC/OS 1.10.0. Install from the DC/OS Service Catalog or use the [quickstart](https://github.com/mesosphere/dcos-kubernetes-quickstart).
+
+### Updated DC/OS Data Services
+
+- Rolling Configuration Update and Upgrades support via the CLI. [enterprise type="inline" size="small" /]
+- Ability to deploy Data Services into Folders to enable multi team deployments. [enterprise type="inline" size="small" /]
+- Ability to deploy to CNI-Based Virtual Networks.
+
+The following updated data services packages are compatible with DC/OS 1.10.
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Could you check and confirm with the SDK team if it is necessary to include the latest link to the release notes for each data service package that are compatible with DC/OS 1.10 OR if the following links are accurate to support the same?

File: pages/1.10/release-notes/1.10.10/index.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+
+  If rebooting a node does not delete old containers and IP/MAC addresses from `etcd` (which over time can cause pool exhaustion), you should set the `--network_cni_root_dir_persist` agent option in the `config.yaml` file to `true`. You should note that changing this flag requires rebooting the agent node or shutting down all container processes running on the node. Because a reboot or shutdown of containers is required, the default value for the `--network_cni_root_dir_persist` agent option is `false`. Before changing this option, you should plan for agent maintenance to minimize any service interruption. If you set this option and reboot a node, you should also unset the `CNI_NETNS` environment variable after rebooting using the CNI plugin `DEL` command so that the plugin cleans up as many resources as possible (for example, by releasing IPAM allocations) and returns a successful response.
+
+## Security
```
Comment (anithamuthu79) [inline pages/1.10/release-notes/1.10.10/index.md]: Add an Enterprise tag.

File: pages/1.10/release-notes/1.10.10/index.md (hunk 5 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: Removed. Simplified.

File: pages/1.10/release-notes/1.10.10/index.md (hunk 7 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: 3 instances fixed

File: pages/1.10/release-notes/1.10.10/index.md (hunk 8 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: Trying to eliminate subjective categories/descriptions, so I would vote not to introduce one for this change. From the user perspective, the change is largely invisible and eliminates a potential warning from an external (not DC/OS) source

File: pages/1.10/release-notes/1.10.10/index.md (hunk 10 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: added.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+
+- Hard CFS CPU limits enabled by default.
+
+  DC/OS 1.10 enforces hard CPU limits with CFS isolation for both the Docker and Universal Container Runtimes. This will give more predictable performance across all tasks but might lead to a slowdown for tasks (and thereby also deployments) who have previously have consumed more CPU cycles than allocated. See [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) for more details.
+
+## Known Issues
+- Upgrade: During upgrade to DC/OS 1.10, there is a brief moment when the DNS resolution does not work. If a health check runs at that moment, it will fail and services will be reported as unhealthy.
+- CORE-1125 - Docker image pull config is re-used.
+- DCOS-16547 - Task state does not update after the agent running it was removed from the cluster.
+- INFINITY-1809 - [Data Svc] DC/OS Service Update / Config Update / Maintenance. [enterprise type="inline" size="small" /]
```
Comment (yishan-lin) [inline pages/1.10/release-notes/1.10.10/index.md]: Where did this come from? @megatmar

File: pages/1.10/release-notes/1.10.10/index.md [hunk 12]
```diff
@@ -0,0 +1,186 @@
...
+  The new command to set up your cluster URL is `dcos cluster setup <dcos_url>`. For details, see [CLI](#cli).
+
+- Hard CFS CPU limits enabled by default.
+
+  DC/OS 1.10 enforces hard CPU limits with CFS isolation for both the Docker and Universal Container Runtimes. This will give more predictable performance across all tasks but might lead to a slowdown for tasks (and thereby also deployments) who have previously have consumed more CPU cycles than allocated. See [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) for more details.
+
+## Known Issues
+- Upgrade: During upgrade to DC/OS 1.10, there is a brief moment when the DNS resolution does not work. If a health check runs at that moment, it will fail and services will be reported as unhealthy.
+- CORE-1125 - Docker image pull config is re-used.
+- DCOS-16547 - Task state does not update after the agent running it was removed from the cluster.
```
Comment (yishan-lin) [inline pages/1.10/release-notes/1.10.10/index.md]: @megatmar Another suspicious ticket...that doesn't belong.

File: pages/1.10/release-notes/1.10.10/index.md (hunk 11 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: this was in a previous version of the release notes (I left the About 1.10 stuff mostly as it has been in the previous releases). If you think I should remove it, I'm happy to do that (having these "holdover" bits is kind of confusing to me, but I opted to stick with the status quo until I hear differently).
Should I remove all or part of what's there? just let me know and I'll fix it.

File: pages/1.10/release-notes/1.10.10/index.md (hunk 12 above)
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: Do you mean all of the known issues (again a holdover from previous release notes).

Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: Hidden content.

File: pages/1.10/release-notes/1.10.10/index.md [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+---
+
+DC/OS 1.10.10 was released on Jaunary 24, 2019.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.10/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Updated Components in DC/OS 1.10.10
+DC/OS 1.10.10 includes the following:
+- Apache Mesos 1.4.3 [change log](https://github.com/apache/mesos/blob/3071ff7/CHANGELOG).
```
[the remark this answers is not in the document]
Comment (lsgunnlsgunn) [reply pages/1.10/release-notes/1.10.10/index.md]: RN 1.10.10 -- Download links:
[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.10/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]

RN 1.10.10 -- Changelog links:
# Updated Components in DC/OS 1.10.10
DC/OS 1.10.10 includes the following:
- Apache Mesos 1.4.3 [change log](https://github.com/apache/mesos/blob/e929d413328e10f6d358899500d5aaedd9d9bc51/CHANGELOG).
- Marathon 1.5.12 [change log](https://github.com/mesosphere/marathon/releases/tag/v1.5.12).
- Metronome 0.4.4 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).

**Pull request merged**